### DISCUSSION TITLE
Unlit material render wrong color

### DISCUSSION THREAD

**Discussion by BCDavid** [google/model-viewer#2428] [Q&A] [answered]
Hello ! Unlit material don't seem to render properly using model viewer.
I tried :
```
environment-image="neutral"
exposure="1"
shadow-intensity="0" 
shadow-softness="0"
```

![issue_unlit_001](https://user-images.githubusercontent.com/12875173/119661958-221e0600-be31-11eb-8a92-ffc91daed2d0.jpg)
Model render on my website with modelviewer: https://www.didierbecet.art/

![issue_unlit_002](https://user-images.githubusercontent.com/12875173/119661964-23e7c980-be31-11eb-8179-67868771e9fc.jpg)
Model render on : https://github.khronos.org/glTF-Sample-Viewer-Release/

Thank you

**Comment by shamsail** [comment]
Try to change environment attribute with **hdr** file

`environment-image="../path/to/src/file.hdr"`

**Reply by BCDavid** [reply]
Thanks for the reply,
I tried with an hdr file, with no change apparently still getting darker color.
Isn't unlit rendering should not be affected by environment image, exposure... ?

**Reply by shamsail** [reply]
you can use exposure from range 0-2 
try to change `exposure="2"`

**Reply by BCDavid** [reply]
I made 2 glb model, one with KHR_materials_unlit and the other without, model viewer doesn't seem to use the extension 
Is there a way to enable the extension ?

Here are the exposure tests :

![unlit_01](https://user-images.githubusercontent.com/12875173/119814078-4be73380-beea-11eb-9109-5c3603b1f810.jpg)
Model render on : https://github.khronos.org/glTF-Sample-Viewer-Release/

![unlit_02](https://user-images.githubusercontent.com/12875173/119814111-55709b80-beea-11eb-88cb-5ee9adf7686c.jpg)
Model render with modelviewer Exposure 1

![unlit_03](https://user-images.githubusercontent.com/12875173/119814173-66211180-beea-11eb-91ed-3cd9f0aaa232.jpg)
Model render with modelviewer Exposure 2

**Reply by shamsail** [reply]
Model viewer has included this, i think so..
just remove environment map from your side as well..

https://github.com/elalish/model-viewer/commit/cbeaf6fd05d2bb3f47e76abb8d2ecc6a37301c09

```
exposure="1"
shadow-intensity="0" 
shadow-softness="0"
```

Just keep this. try it

**Reply by BCDavid** [reply]
Environment map is removed but no change so far.
Unlit material seem to render fine on all viewers except modelviewer, I can't find if the issue come from my model or not.

Here are my gltf files if it can help :
https://assets.didierbecet.art/GLTF/med_ray_002.gltf
https://assets.didierbecet.art/GLTF/med_ray_002.bin

**Comment by BCDavid** [comment, marked as answer]
I found a workaround by reducing the texture contrast by -25% / -50%.

![issue_001](https://user-images.githubusercontent.com/12875173/119859209-78fd0b80-bf15-11eb-99c4-87fdb5053430.jpg)

**Reply by elalish** [reply]
This is interesting; it looks like the issue might be tone-mapping being applied to the unlit result? That could be a bug. In the sample renderer I believe they have tone-mapping options; do those have any effect?

**Reply by elalish** [reply]
Fixed in #3152.
